spec/acceptance/iis_application_pool_spec.rb

```diff
@@ -50,8 +50,8 @@
         
         include_context 'with a puppet resource run'
         puppet_resource_should_show('ensure', 'present')
-        puppet_resource_should_show('managedpipelinemode', 'Classic')
-        puppet_resource_should_show('managedruntimeversion', 'v4.0')
+        puppet_resource_should_show('managed_pipeline_mode', 'Classic')
+        puppet_resource_should_show('managed_runtime_version', 'v4.0')
```

Comment:
puppet resource should show all the attributes, this should be testing for them.